### DISCUSSION TITLE
Add different type of goals for numerical habits

### DISCUSSION THREAD

**Discussion by Stiralbios** [iSoron/uhabits#949] [Feature Requests] [answered]
Numerical habbits have goals but the goal is always to go above a threshold.
It would be nice if we could personnalize this to consider the goal as reached if we stay below a threshold or if we get between two thresholds.

 This could be usefull in different situations like counting calories for a diet or trying to reduce an habbit by counting how many times we do it in one day.

**Comment by hiqua** [comment]
For the maximum value, from what I understand the main problem for now is how to compute the score.

**Comment by KristianTashkov** [comment]
Wasn't there a `AT_MOST` numerical type in the app code? I guess it's not connected and implemented yet? Is there a feature request beside this one that tracks that part of numerical habits since the main one got completed?

**Reply by hiqua** [reply]
> I guess it's not connected and implemented yet?

Exactly, see the message from @iSoron sent in the chat room on August 4th.

**Comment by plibither8** [comment]
Seconding this! Particularly useful for numerical un-habits, for example, f I want to bring down the frequency of a bad habit to 0 per day.

**Comment by iSoron** [comment, marked as answer]
This has been implemented by @KristianTashkov (see #1101) and should become available in Loop 2.1.
